### ISSUE TITLE
Error saving the resource

### ISSUE THREAD

**Issue by marianvlad** [dcasia/nova-json-wrapper#7]
I'm using Nova 3 and when I save the resource I get "json_decode() expects parameter 1 to be string, array given" [JsonWrapper.php#L104](https://github.com/dcasia/nova-json-wrapper/blob/master/src/JsonWrapper.php#L104)

**Comment by jornwildenbeest** [comment]
Same here

**Comment by mennotempelaar** [comment]
#8 did not fix the json_decode error

**Comment by milewski** [comment]
I think this may happen because the attribute is being cast to an array? is there any $casts=[] set on any of your code? ..

**Comment by mennotempelaar** [comment]
@milewski Yea there is. It's on the Readme.md file to add the cast to your JsonWrapper column.

And you need the cast to use it within your application I guess or to fill the fields within the JsonWrapper.

I guess the result from the getOriginal method has changed? This should return the uncasted value of the attribute right?

**Comment by milewski** [comment]
Yes, it should have to be a string.. that was supposed to get the original data unmodified as in:

```php
$model->property // [a,b,c] array
$model->isDirty() // false
$model->property = [1,2,3]

$model->getOriginal() // "[a,b,c]" string
$model->property // [1,2,3] array
$model->isDirty() // true
```

Maybe laravel 7 changed this behavior.. and now getOriginal will run the casts on it ...

**Comment by milewski** [comment]
Well.. that is exacly what happened: 

https://laravel.com/docs/7.x/upgrade#factory-types

![image](https://user-images.githubusercontent.com/2874967/84142565-94ae9e00-aa87-11ea-8d8d-549c2eda48f4.png)

**Comment by milewski** [comment]
I think the fix is easy.. just need to change it to `getRawOriginal` :D can you PR it

**Comment by mennotempelaar** [comment]
Yea I've forked another project to edit this, but I think you can change this much faster :D

**Comment by mennotempelaar** [comment]
AH k ;) I'll PR this

**Comment by milewski** [comment]
Thanks published as `v0.1.6` 🎉
